package.json

```diff
@@ -103,6 +100,7 @@
     "eslint-plugin-local": "1.0.0",
     "eslint-plugin-prefer-let": "3.0.1",
     "glob": "10.3.10",
+    "isolated-vm": "5.0.1",
```

Comment:
Make sure to revert this, this is the way it is because it breaks a lot of external developers on various platforms.